azure-pipelines.yml

```diff
@@ -116,7 +116,7 @@ stages:
     parameters:
       enablePublishBuildArtifacts: true
       enablePublishTestResults: true
-      enablePublishUsingPipelines: $(_PublishUsingPipelines)
+      enablePublishUsingPipelines: ${{ variables._PublishUsingPipelines }}
```

Comment:
Need to also update https://github.com/aspnet/AspNetCore-Tooling/blob/b3668389b72c1faccb198d3efc4244a0f1049085/azure-pipelines.yml#L148 later in this file.

Reply:
That one isn't affected by the problem that this one in line 119 is. Only assignments that are later used in template expressions is.

Reply:
:ok: the other one doesn't _need_ to change. The inconsistency will hopefully not cause someone to change this line back.